Formula/gcc.rb

```diff
@@ -28,16 +27,6 @@ class Gcc < Formula
   # GCC bootstraps itself, so it is OK to have an incompatible C++ stdlib
   cxxstdlib_check :skip
 
-  # Patch for Xcode bug, taken from https://gcc.gnu.org/bugzilla/show_bug.cgi?id=89864#c43
-  # This should be removed in the next release of GCC if fixed by apple; this is an xcode bug,
-  # but this patch is a work around committed to GCC trunk
-  if MacOS::Xcode.version >= "10.2"
-    patch do
-      url "https://raw.githubusercontent.com/Homebrew/formula-patches/master/gcc/8.3.0-xcode-bug-_Atomic-fix.patch"
-      sha256 "33ee92bf678586357ee8ab9d2faddf807e671ad37b97afdd102d5d153d03ca84"
-    end
-  end
```

Comment:
LGTM

Reply:
I spoke too soon. It looks like they're targeting 9.2 for the fix... I'll try to attend to this in ~ 2 hours

Reply:
OK, I can confirm that this is in 9.1. Sorry for the confusion. Someone updated the bug report to say it was targeting 9.2 which was the source of my confusion.